xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.84f.xml

```diff
@@ -54,14 +100,17 @@
       <revisionDesc>
          <change when="2010-09-29" who="CDS">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="EM">More tweaks for bib only inscriptions</change>
+         <change when="2020-02-12" who="SPapandonatou">added metadata and transcription</change>
       </revisionDesc>
    </teiHeader>
    <!--facsimile element goes here--><text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
+           <lb n="1"/>ΕΥΤΥΧ<gap reason="lost" extent="unknown"/> 
```

Comment:
I suspect we're missing an eta in the part of the text that's missing (the label would be pretty standard, I think), but if it doesn't look like that would fit we can leave what you have.